README.md

```diff
@@ -193,7 +193,9 @@ app.listen(80, function () {
 
 * `origin`: Configures the **Access-Control-Allow-Origin** CORS header. Possible values:
   - `Boolean` - set `origin` to `true` to reflect the [request origin](http://tools.ietf.org/html/draft-abarth-origin-09), as defined by `req.header('Origin')`, or set it to `false` to disable CORS.
-  - `String` - set `origin` to a specific origin. For example if you set it to `"http://example.com"` only requests from "http://example.com" will be allowed.
+  - `String` - set `origin` to a specific origin. For example, if you set it to
+    - `"http://example.com"` only requests from "http://example.com" will be allowed.
+    - `"*"` for all domains to be allowed. This is UNSAFE and not recommended.
```

Comment:
> This is UNSAFE and not recommended.

Can you elaborate on this, please? If it is unsafe and not recommended, then why would we leave this as a feature at all? I'm interested in learning more if you can point to some resources that explain where this statement comes from.

Reply:
Hi - Based on these couple of documentation online - 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin
https://stackoverflow.com/questions/12001269/what-security-risks-exist-when-setting-access-control-allow-origin-to-accept-all#:~:text=on%20this%20post.-,Access%2DControl%2DAllow%2DOrigin%3A%20*%20is%20totally%20safe,auth%2C%20and%20TLS%20client%20certificates
PLs advise.

Reply:
I don't see anything about it being unsafe in your first link and the second link says both it is fine and not fine and doesn't seem conclusive, as it would depend on a lot of factors. The answer also looks like it is CORS in general and nothing specific to this module. The docs of this module should only be about using this module -- not all the details and information about CORS itself.

Reply:
sounds good. Let me fix it.

Reply:
@dougwilson - pls review